connect/cli/plugins/project/report/templates/bootstrap/${project_slug}/tests/conftest.py.j2

```diff
@@ -47,12 +51,64 @@ def _parse_qs(url):
     return query, ordering, select
 
 
+def _mock_kwargs_generator(response_iterator, url):
+    res = next(response_iterator)
+
+    query, ordering, select = _parse_qs(url)
+    if res.query:
+        assert query == res.query, 'RQL query does not match.'
+    if res.ordering:
+        assert ordering == res.ordering, 'RQL ordering does not match.'
+    if res.select:
+        assert select == res.select, 'RQL select does not match.'
+    mock_kwargs = {
+        'match_querystring': False,
+    }
+    if res.count is not None:
+        end = 0 if res.count == 0 else res.count - 1
+        mock_kwargs['status'] = 200
+        mock_kwargs['headers'] = {'Content-Range': f'items 0-{end}/{res.count}'}
+        mock_kwargs['json'] = []
+
+    mock_kwargs.update(_value_arg_validation(res))
+    return mock_kwargs
+
+
+def _value_arg_validation(res):
+    result = {}
+    if isinstance(res.value, Iterable):
+        count = len(res.value)
+        end = 0 if count == 0 else count - 1
+        result['status'] = 200
+        result['json'] = res.value
+        result['headers'] = {
+            'Content-Range': f'items 0-{end}/{count}',
+        }
+    elif isinstance(res.value, dict):
+        result['status'] = res.status or 200
+        result['json'] = res.value
+    elif res.value is None:
+        if res.exception:
+            result['body'] = res.exception
+        else:
+            result['status'] = res.status
+    else:
+        result['status'] = res.status or 200
+        result['body'] = str(res.value)
+    return result
+
+
 @pytest.fixture
 def response():
     with responses.RequestsMock() as rsps:
         yield rsps
 
 
+@pytest.fixture
```

Comment:
What is this ? Afaik it is need in extension since the extension constructor requires a logger.